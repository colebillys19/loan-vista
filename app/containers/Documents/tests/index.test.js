import React from 'react';
import { render } from 'react-testing-library';
import { ConnectedRouter } from 'connected-react-router';
import { Provider } from 'react-redux';

import configureStore from 'configureStore';
import history from 'utils/history';
import { Documents } from '../index';

// Create redux store with history
const initialState = {};
const store = configureStore(initialState, history);

const WrappedDocuments = (props) => (
  <Provider store={store}>
    <ConnectedRouter history={history}>
      <Documents {...props} />
    </ConnectedRouter>
  </Provider>
);

describe('<Documents />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<WrappedDocuments dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it.skip('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<WrappedDocuments />);
    expect(firstChild).toMatchSnapshot();
  });
});
