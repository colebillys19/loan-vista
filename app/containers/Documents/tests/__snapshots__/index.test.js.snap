// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Documents /> Should render and match the snapshot 1`] = `
<section
  aria-labelledby="documents-tab"
  class="TabWrapper-xht0pf-0 hVyZPG"
  id="documents-view"
>
  <form
    class="styledComponents__StyledForm-sc-6nnwf7-2 kpDAEG"
  >
    <span
      class="styledComponents__DisableWrapper-sc-6nnwf7-0 emJxZD"
    >
      <label
        class="styledComponents__StyledLabel-sc-6nnwf7-4 fnnyyM"
        for=""
      >
        from
      </label>
      <span
        class="styledComponents__DatePickerWrapper-sc-1wk5y4r-1 sdcXD"
      >
        <div
          class="MuiFormControl-root MuiTextField-root styledComponents__StyledKeyboardDatePicker-sc-1wk5y4r-2 doVNdr styledComponents__StyledTextField-sc-1wk5y4r-3 eubNBY"
          title=""
        >
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
              placeholder="date"
              type="text"
              value=""
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
            >
              <button
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          
        </div>
        <div
          class="styledComponents__ErrorIndicator-sc-1wk5y4r-0 oDsFE"
        />
      </span>
    </span>
    <span
      class="styledComponents__DisableWrapper-sc-6nnwf7-0 emJxZD"
    >
      <label
        class="styledComponents__StyledLabel-sc-6nnwf7-4 fnnyyM"
        for=""
      >
        to
      </label>
      <span
        class="styledComponents__DatePickerWrapper-sc-1wk5y4r-1 sdcXD"
      >
        <div
          class="MuiFormControl-root MuiTextField-root styledComponents__StyledKeyboardDatePicker-sc-1wk5y4r-2 doVNdr styledComponents__StyledTextField-sc-1wk5y4r-3 eubNBY"
          title=""
        >
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
              placeholder="date"
              type="text"
              value=""
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
            >
              <button
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          
        </div>
        <div
          class="styledComponents__ErrorIndicator-sc-1wk5y4r-0 oDsFE"
        />
      </span>
    </span>
    <span
      class="styledComponents__DisableWrapper-sc-6nnwf7-0 emJxZD"
    >
      <label
        class="styledComponents__StyledLabel-sc-6nnwf7-4 fnnyyM"
        for=""
      >
        or
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline styledComponents__StyledSelect-sc-6nnwf7-6 dzbarT"
      >
        <div
          aria-haspopup="listbox"
          class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
          role="button"
          tabindex="0"
        >
          date range
        </div>
        <input
          aria-hidden="true"
          class="MuiSelect-nativeInput"
          tabindex="-1"
          value="0"
        />
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
    </span>
    <span
      class="styledComponents__DisableWrapper-sc-6nnwf7-0 emJxZD"
    >
      <label
        class="styledComponents__StyledLabel-sc-6nnwf7-4 fnnyyM"
        for=""
      >
        and/or
      </label>
      <div
        class="MuiFormControl-root MuiTextField-root styledComponents__StyledTextField-sc-6nnwf7-7 bukyEO"
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            placeholder="keyword search"
            type="text"
            value=""
          />
        </div>
      </div>
    </span>
    <button
      aria-label="delete"
      class="MuiButtonBase-root MuiIconButton-root styledComponents__StyledIconButton-sc-6nnwf7-3 LvLiT"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiIconButton-label"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          style="height: 2.5rem; width: 2.5rem;"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="transparent"
          />
          <path
            d="M17.65 6.35c-1.63-1.63-3.94-2.57-6.48-2.31-3.67.37-6.69 3.35-7.1 7.02C3.52 15.91 7.27 20 12 20c3.19 0 5.93-1.87 7.21-4.56.32-.67-.16-1.44-.9-1.44-.37 0-.72.2-.88.53-1.13 2.43-3.84 3.97-6.8 3.31-2.22-.49-4.01-2.3-4.48-4.52C5.31 9.44 8.26 6 12 6c1.66 0 3.14.69 4.22 1.78l-1.51 1.51c-.63.63-.19 1.71.7 1.71H19c.55 0 1-.45 1-1V6.41c0-.89-1.08-1.34-1.71-.71l-.64.65z"
            fill="#7b68ee"
          />
        </svg>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <span
      class="styledComponents__DisableWrapper-sc-6nnwf7-0 emJxZD"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-6nnwf7-1 fyzjVE styledComponents__StyledButton-sc-86ulmm-0 exBHac"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiButton-label"
        >
          find
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </span>
  </form>
  <div
    class="styledComponents__TableWrapper-qh5wjg-8 hoogGb"
  >
    <div
      class="styledComponents__GradientBorder-sc-16ewd9t-0 cERlnS"
    />
    <div
      class="styledComponents__GradientBorder-sc-16ewd9t-0 bWdJUA"
    />
    <div
      class="styledComponents__GradientBorder-sc-16ewd9t-0 cPJiSx"
    />
    <div
      class="styledComponents__GradientBorder-sc-16ewd9t-0 bJrMFD"
    />
    <table
      class="styledComponents__StyledTable-qh5wjg-1 hxxaBn"
    >
      <thead
        class="styledComponents__StyledTableHead-qh5wjg-4 styledComponents__CustomTableHead-sc-13flwk-0 ihNRbH"
      >
        <tr
          class="styledComponents__StyledTableRow-qh5wjg-6 hCqVZR"
        >
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1qurbx9-0 fqrnhO MuiButton-disableElevation"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                date sent
                <span
                  class="MuiButton-endIcon MuiButton-iconSizeMedium"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    style="height: 0.8rem; width: 0.8rem;"
                    viewBox="7 7 10 10"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M7 10l5 5 5-5z"
                      fill="#888888"
                    />
                  </svg>
                </span>
              </span>
            </button>
          </th>
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            time sent
          </th>
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1qurbx9-0 fqrnhO MuiButton-disableElevation"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                type
                <span
                  class="MuiButton-endIcon MuiButton-iconSizeMedium"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    style="height: 0.8rem; width: 0.8rem;"
                    viewBox="7 7 10 10"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M7 10l5 5 5-5z"
                      fill="#888888"
                    />
                  </svg>
                </span>
              </span>
            </button>
          </th>
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1qurbx9-0 fqrnhO MuiButton-disableElevation"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                from
                <span
                  class="MuiButton-endIcon MuiButton-iconSizeMedium"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    style="height: 0.8rem; width: 0.8rem;"
                    viewBox="7 7 10 10"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M7 10l5 5 5-5z"
                      fill="#888888"
                    />
                  </svg>
                </span>
              </span>
            </button>
          </th>
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            desc
          </th>
          <th
            class="styledComponents__StyledTableHeader-qh5wjg-5 gupKNG"
            scope="col"
          >
            pdf
          </th>
        </tr>
      </thead>
      <tbody
        class="styledComponents__StyledTableBody-qh5wjg-2 iHTrkK"
      >
        <tr
          class="styledComponents__StyledTableRow-qh5wjg-6 hCqVZR"
        >
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            date sent
          </td>
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            time sent
          </td>
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            type
          </td>
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            from
          </td>
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            desc
          </td>
          <td
            class="styledComponents__StyledTableData-qh5wjg-3 hpnAfB"
          >
            <button
              class="MuiButtonBase-root styledComponents__StyledButton-sc-9o47wt-0 fgJPGq"
              tabindex="0"
              type="button"
            >
              download
            </button>
          </td>
        </tr>
      </tbody>
      <thead>
        <tr>
          <td
            class="styledComponents__TableFooterCell-qh5wjg-7 jVHZPT"
          >
            hidden text
          </td>
        </tr>
      </thead>
    </table>
  </div>
</section>
`;
